### PR TITLE
W-14970623 - Adding RPA Builder 6.5.6 RNs

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== February 6, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The Tomcat dependency is upgraded to version 9.0.85 to address reported security vulnerabilities.
```
Comment (justus-schach) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc]: Tomcat upgrade was already contained in RPA Builder Version 6.5.4

**Review by justus-schach — DISMISSED** [review]
I guess the Tomcat upgrade hint was doubled.

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc (hunk 1 above)
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc]:
```suggestion
```

Comment (Cristian-Venticinque) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc]: You are correct, this was a duplicate, just removed it. Thanks!

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
+= RPA Builder 6.5.6 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== February 6, 2024
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc]:
```suggestion
== February 7, 2024
```

**Review by failup — DISMISSED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+// == Fixed Issues
+
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="5a,1a"]
+|===
+| Issue Resolution | ID
+| The Math Operations action step no longer takes more time than expected to perform operations. | W-14970079
+| RPA Builder no longer crashes when you configure the Select Case action step to compare an alphanumeric value that contains an underscore character ('_') and this character is selected as the comparison value. | W-14865141
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.6-release-notes.adoc]:
```suggestion
| RPA Builder no longer crashes when you configure the Select Case action step to compare an alphanumeric value that contains an underscore character (`_`) and this character is selected as the comparison value. | W-14865141
```

**Review by failup — APPROVED** [review]

**Pull request merged**